### PR TITLE
Added forking step to instructions

### PR DESCRIPTION
**Pull request by vatri**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | https://github.com/symfony/recipes-contrib/issues/678

Added forking step to instructions as discussed in the above issue.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -10,6 +10,13 @@ the Symfony Core Team. For contributed recipes, see the `contrib repository`_.
 Creating Recipes
 ----------------
 
+1. Clone a repository
```
Comment (bocharsky-bw) [inline README.rst]: Hm, technically it's not required to _clone_ the repo, you can use GitHub GUI for this

Comment (vatri) [reply README.rst]: @bocharsky-bw This is error , I wanted to write "fork" instead "clone". I have fixed it now.

**Comment by bocharsky-bw** [comment]
Fairly speaking, I agree with @maxhelias - it seems too obvious for me. IIRC everywhere in Symfony components, we just say "send Pull Requests in the main Symfony repository" without explaining how to do this, because for us it's done upstream by GitHub. The same here I think

**Comment by vatri** [comment]
@maxhelias You might be right.

I was reading some tutorial so I was confused of how and where recipes are being created.

**Comment by fabpot** [comment]
Let's close for the reasons explained by some commenters

**Pull request closed without merging**